docker/packager/binary/build.sh

```diff
@@ -29,8 +29,21 @@ env
 
 if [ -n "$MAKE_DEB" ]; then
   rm -rf /build/packages/root
+  if [ -z "$SANITIZER" ]; then
+    # We need to check if clickhouse-diagnostics is fine and build it
+    (
+      cd /build/programs/diagnostics
+      make test-no-docker
+      GOARCH="${DEB_ARCH}" CGO_ENABLED=0 make VERSION="$CLICKHOUSE_VERSION_STRING" build
```

Comment:
Can we pass in ldflags here with the version? It ensures the binary can compile in the version and will report the correct version with the user does `clickhouse-diagnostics version`. see https://github.com/ClickHouse/clickhouse-diagnostics/blob/main/Makefile#L8.
I think we can align with the clickhouse version?

Reply:
ah nvm i see you do below.